packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js

```diff
@@ -40,16 +42,28 @@ const ContentItem = ({ copy, heading, image }) => (
         data-autoid={`${stablePrefix}--content-item__copy`}
         className={`${prefix}--content-item__copy`}
         dangerouslySetInnerHTML={{
-          __html: markdownToHtml(copy),
+          __html: markdownToHtml(copy, { bold: false }),
         }}></div>
     )}
+    {cta && <CTA style="text" type={cta.type} copy={cta.copy} />}
   </div>
 );
 
 ContentItem.propTypes = {
+  cta: PropTypes.shape({
+    type: PropTypes.string,
+    copy: PropTypes.string,
+  }),
   copy: PropTypes.string,
   heading: PropTypes.string,
-  image: PropTypes.object,
+  image: PropTypes.shape({
+    images: PropTypes.shape({
+      src: PropTypes.string,
+      minWidth: PropTypes.string,
+    }),
+    alt: PropTypes.string,
+    defaultImage: PropTypes.string,
+  }),
```

Comment:
I'm not sure I would worry about the image, or cta shape here since that's being defined by those components.

Reply:
Hmm, we might want to if we are adding restrictions to certain props on a wrapper/decorator level, right? e.g. no passing of certain props...

Reply:
I don't think we're doing much restriction outside of `cta.style` for this component. I feel like if you wanted to get more specific than object you could use `PropTypes.instanceOf()`. In this given scenario I wouldn't be worried about style because it'll just be ignored since the component itself is hard coding it.

```javascript
ContentItem.propTypes = {
  cta: PropTypes.instanceOf(CTA),
  image: PropTypes.instanceOf(Image),
  ...
}
```

You could use `exact` if you want to get a warning thrown when cta.styles is defined, but you'll still have to keep up with those props if anything changes down the road. There might be a scenario for one of our decorators, but I'm personally not seeing it here.

Reply:
I like `instanceOf`.

Reply:
ok, changing to `instanceOf`!